scripts/release/download-gh-artifacts.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+RELEASE=$1
+SIGNER=$2
+
+echo "RELEASE: $RELEASE"
+echo "SIGNER: $SIGNER"
+
+for com in gh gpg curl sha256sum ; do
+	command -V ${com} >/dev/null 2>&1
+done
+
+[ ! -e "gh-release-artifacts/${RELEASE}" ]
+
+mkdir -p "gh-release-artifacts/${RELEASE}"
+cd "gh-release-artifacts/${RELEASE}"
+
+# github
+gh release download $RELEASE
+
+# cirrus
+curl -L -o haskell-language-server-${RELEASE}-x86_64-freebsd.tar.xz \
+	"https://api.cirrus-ci.com/v1/artifact/github/haskell/haskell-language-server/build/binaries/out/haskell-language-server-${RELEASE}-x86_64-freebsd.tar.xz?branch=${RELEASE}"
+
+sha256sum haskell-language-server-* > SHA256SUMS
+gpg --detach-sign -u "${SIGNER}" SHA256SUMS
+
```

Comment:
This doesn't include the bit about uploading to `downloads.haskell.org` or clearing the CDN caches. Either we need to add those (preferably adapting `release/upload.sh`), or `release/upload.sh` can stay as it is, and we remove the hashing and signing bits from this script.

Reply:
Sure, I can remove the hashing and signing bits. I don't really automate bindist uploads myself, so I'm not sure how you want to do that.